src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -46,7 +48,7 @@ public EditCommand parse(String args) throws ParseException {
         }
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS,
-                PREFIX_REMARK, PREFIX_PRIORITY);
+                PREFIX_REMARK, PREFIX_PRIORITY, PREFIX_DATE_OF_BIRTH);
```

Comment:
Should the prefix for income also be checked for no duplicates here?

Reply:
Thanks for catching later